pkg/registry/bundle.go

```diff
@@ -41,7 +41,7 @@ type Bundle struct {
 	Package      string
 	Channels     []string
 	BundleImage  string
-	csv          *ClusterServiceVersion
+	Csv          *ClusterServiceVersion
```

Comment:
why did this change to public?

Reply:
https://github.com/operator-framework/operator-registry/pull/448/files#diff-08f655f8aa9db090b9d078c7c11450a3R141 we need to unpack the images here and in order to know which images to unpack we need to access these fields outside the registry package

Reply:
You shouldn't need `Csv.GetName` - that's what `Name` is, which is already public

Reply:
gotcha! will revert that change then